File: seed/swift-sdk/pagination/custom-pager-with-exception-handler/Tests/Snippets/Example21.swift
import Foundation
import Pagination

enum Example21 {
    static func snippet() async throws {
        let client = PaginationClient(
            baseURL: "https://api.fern.com",
            token: "<token>"
        )

        _ = try await client.users.listWithOffsetStepPagination(
            page: 1,
            limit: 1,
            order: .asc
        )
    }
}

File: seed/csharp-sdk/pagination/page-index-semantics/Snippets/Example21.cs
using SeedPagination;

public partial class Examples
{
    public async Task Example21() {
        var client = new SeedPaginationClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Users.ListWithOffsetStepPaginationAsync(
            new SeedPagination.ListUsersOffsetStepPaginationRequest {
                Page = 1,
                Limit = 1,
                Order = SeedPagination.Order.Asc
            }
        );
    }

}
